schema-engine/sql-migration-tests/tests/migrations/db_execute.rs

```diff
@@ -32,7 +32,7 @@ fn db_execute_happy_path_with_literal_url() {
     "#;
 
     // Execute the command.
-    let generic_api = migration_core::migration_api(None, None).unwrap();
+    let generic_api = schema_core::schema_api(None, None).unwrap();
```

Comment:
is `generic_api` supposed to reference the set of functionality across all connectors?

Reply:
Yep. So that's the JSON RPC API that can do introspection, migrations, diffing etc. for all the connectors.